python/mxnet/library.py

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# coding: utf-8
+"""Library management API of mxnet."""
+from __future__ import absolute_import
+import ctypes
+import os
+from .base import _LIB, check_call, MXNetError
+
+def load(path):
+    """Loads library dynamically.
+
+    Parameters
+    ---------
+    path : Path to library .so/.dll file
+
+    Returns
+    ---------
+    void
+    """
+    #check if path exists
+    if not os.path.exists(path):
+        raise MXNetError("load path %s does NOT exist" % path)
+    #check if path is an absolute path
```

Comment:
Why does it have to be an absolute path? Can we clarify? What if the library is installed in /usr/lib?

Reply:
Dynamic loader (dlopen) requires the path to be absolute.

Reply:
So can't we make it absolute programmatically in a Python level as a convenience for the user?

Reply:
Thanks for the suggestion @ptrendx. We thought about this. We could, but if the user passes in "./libmyacc.so" and Python is running in a different dir, we're going to make the wrong absolute path "/user/home/libmyacc.so". 

So initially we were thinking that it just makes more sense to require the user to provide that as input. It would avoid confusion around "how is this path getting created".